api/src/lib/interface.ts

```diff
@@ -76,6 +76,7 @@ export interface CreatePublicationRequestBody {
     content?: string;
     language?: Languages;
     fundersStatement: string;
+    conflictOfInterestStatus?: boolean | null;
```

Comment:
Maybe, but perhaps I got confused the otherday, but the payload that is sent in createPublication contains the conflictOfInterestStatus:

```typescript
const payload = {
    type: 'PROBLEM',
    title,
    ethicalStatement: false,
    conflictOfInterestStatus: 0,
    language: 'en',
    licence: 'CC_BY',
    content: content || title,
    description: content
};
```

Which was not setting it to false, therefore the function to set the publication to live does not work, as once of the requirements is to declare this, it can't be null on a LIVE publication, so I added this in order to resolve that issue.

I assume it should be done a different way, perhaps with another API call to update the publication after it been created in draft, perhaps on the update endpoint?